defaults/main.yml

```diff
@@ -55,3 +55,15 @@ keepalived_bind_on_non_local: False
 # Example:
 #keepalived_global_defs:
 #  - enable_script_security
+
+# Set location of keepalived daemon options file path
+# For Debian based systems it's usually /etc/default/keepalived
+# For RedHat based systems it's usually /etc/sysconfig/keepalived
+keepalived_daemon_options_file_path: ""
```

Comment:
if keepalived_daemon_options_file_path is defined here, its condition in tasks is always true.

Reply:
keepalived_daemon_options_file_path seems distro specific, and should be moved to vars, providing a good "default"